demo/dlb-init.sh

```diff
@@ -46,4 +47,8 @@ enable_and_configure_vfs() {
 
 # use first dlb device to configure a vf
 DEVPATH=$(realpath /sys/bus/pci/drivers/dlb2/????:??:??\.0 | head -1)
-enable_and_configure_vfs "$DEVPATH"
+PCI_DEV=$(cat "$DEVPATH"/device 2> /dev/null)
+if [ "$PCI_DEV" == "$DLB_DEVICE_PCI_ID" ]; then
```

Comment:
This checks one PCI ID only but what if others need to be checked too? It's always good to make it generic.

But I feel the check is useless here. Could we just use `! test -w "$sriov_numvfs_path"` but revisit the exit code

Reply:
I was thinking, we may add for loop that will contain these lines.
But, yeah, let me just fix as you suggested.!

Reply:
so you mean, to change exit 1 to exit 0 and fix the error message, right?